packages/react-ui/src/widgets/FeatureSelectionWidgetUI/FeatureSelectionUIGeometryChips.js

```diff
@@ -1,16 +1,18 @@
+import { Cancel } from '@mui/icons-material';
 import { Box, Chip, List, ListItem, Tooltip, styled } from '@mui/material';
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useState } from 'react';
 
 const ChipList = styled(List)(({ theme: { spacing } }) => ({
   display: 'flex',
   alignItems: 'center',
   margin: 0,
-  padding: spacing(0, 1),
+  padding: 0,
```

Comment:
Removing this padding the storybook now looks a bit weird
![Captura de pantalla 2023-06-14 a las 17 18 38](https://github.com/CartoDB/carto-react/assets/6652814/84365257-db7b-485d-b9bc-c447187a5a1a)

Reply:
Fixed https://github.com/CartoDB/carto-react/pull/708/commits/42e895c5efaa33308c60f8179f15229616789a90